models/src/main/protobuf/CasperMessage.proto

```diff
@@ -178,11 +178,12 @@ message BodyProto {
   repeated ProcessedDeployProto deploys      = 2;
   repeated ProcessedSystemDeployProto systemDeploys = 3;
   bytes                    extraBytes   = 4;
-  repeated RejectedDeployProto rejectedDeploys = 5;
+  repeated MergingDeployStatusProto mergingDeployStatuses = 5;
 }
 
-message RejectedDeployProto{
+message MergingDeployStatusProto{
```

Comment:
Why do we need to define the same message on two places? Here and in DeployServiceCommon.proto.